### DISCUSSION TITLE
v4.14.0

### DISCUSSION THREAD

**Discussion by luu-alex** [ethereum/web3.js#7344] [Announcements]
## What's Changed
* Fix CHANGELOG for PR 7239 by @danforbes in https://github.com/web3/web3.js/pull/7271
* Fix Typos in Docs by @danforbes in https://github.com/web3/web3.js/pull/7283
* Revert CHANGELOG Changes from #7271 by @danforbes in https://github.com/web3/web3.js/pull/7274
* Add Note About create-hardhat-web3 to Hardhat Guide by @danforbes in https://github.com/web3/web3.js/pull/7281
* fix padRight validation failure on large uint by @Muhammad-Altabba in https://github.com/web3/web3.js/pull/7265
* Ok/6984 update e2e network tests by @avkos in https://github.com/web3/web3.js/pull/7276
* feat: make getEthereumjsTxDataFromTransaction not trim extra fields by @nicolasbrugneaux in https://github.com/web3/web3.js/pull/7282
* 7136 decodeparams update by @jdevcs in https://github.com/web3/web3.js/pull/7288
* Update FUNDING.json by @Redoudou in https://github.com/web3/web3.js/pull/7304
* Fix e2e test error "project ID request rate exceeded" by @avkos in https://github.com/web3/web3.js/pull/7290
* fix: correct type and documentation for `baseFeePerGas` at `web3.eth.getFeeHistory` by @Muhammad-Altabba in https://github.com/web3/web3.js/pull/7291
* Restructure Docs by @danforbes in https://github.com/web3/web3.js/pull/7298
* Add Support for Nethermind and Besu 'syncing' Subscription Payload Format by @chebykin in https://github.com/web3/web3.js/pull/7306
* Fix Typos by @danforbes in https://github.com/web3/web3.js/pull/7310
* Export EIP-6963 Types by @danforbes in https://github.com/web3/web3.js/pull/7270
* Add Gas Estimation Guide by @danforbes in https://github.com/web3/web3.js/pull/7319
* add migration websocket doc changes by @luu-alex in https://github.com/web3/web3.js/pull/7318
* add ignoregaspricing config by @luu-alex in https://github.com/web3/web3.js/pull/7320

## New Contributors
* @Redoudou made their first contribution in https://github.com/web3/web3.js/pull/7304
* @chebykin made their first contribution in https://github.com/web3/web3.js/pull/7306

**Full Changelog**: https://github.com/web3/web3.js/compare/v4.13.0...v4.14.0

<hr /><em>This discussion was created from the release <a href='https://github.com/web3/web3.js/releases/tag/v4.14.0'>v4.14.0</a>.</em>

**Comment by rioyuda02** [comment]
I will try to this update,
